glean-core/android/src/test/java/mozilla/telemetry/glean/utils/DateUtilsTest.kt

```diff
@@ -3,10 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+package mozilla.telemetry.glean.utils
```

Comment:
huh, what happened in this file?

Reply:
It was missing the package declaration and Android Studio reported a warning, so I fixed it (see the related "BONUS:" commit message